packages/components/src/styles/components/icon.scss

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+.flight-icon {
```

Comment:
don't forget that now the namespace should be `hds-icon`

(plus, we have to understand which of the previous `flight-icon` class names we want to preserve, because they're [used in the consumers codebases](https://github.com/search?q=flight-icon++user%3Ahashicorp+-repo%3Ahashicorp%2Fdesign-system+-repo%3Ahashicorp%2Fdesign-system-metrics&type=code&ref=advsearch))